### ISSUE TITLE
Error when trying to bundle with browserify

### ISSUE THREAD

**Issue by aimjel** [talmobi/yt-search#80]
Error: Can't walk dependency graph: Cannot find module './util.js' from 'C:\Users\angel\AppData\Roaming\projct\node_modules\yt-search\dist\yt-search.js

**Comment by talmobi** [comment]
@aimjel Hello there~

Try with `browserify --node --no-bundle-external --standalone`

**Comment by aimjel** [comment]
> @aimjel Hello there~
> 
> Try with `browserify --node --no-bundle-external --standalone`

Then it wont bundle the yt-search module

**Comment by talmobi** [comment]
@aimjel yes -- there should be no need to bundle in third party code -- this is a server side library 🙇
